{"nodes":[{"pos":[687,813],"content":"This exception is typically thrown on the client when a channel is terminated due to server closing the associated connection.","needQuote":true,"needEscape":true,"nodes":[{"content":"This exception is typically thrown on the client when a channel is terminated due to server closing the associated connection.","pos":[0,126]}]},{"pos":[825,1402],"content":"This exception derives from <xref:System.ServiceModel.CommunicationException> that represents a class of recoverable errors that may be thrown during communication between endpoints and which robust [!INCLUDE[indigo1](~/add/includes/ajax-current-ext-md.md)] client and service applications are expected to handle. To prevent the more generic <xref:System.ServiceModel.CommunicationException> handler from catching the more specific <xref:System.ServiceModel.ActionNotSupportedException>, catch this exception prior to handling <xref:System.ServiceModel.CommunicationException>.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"This exception derives from &lt;xref:System.ServiceModel.CommunicationException&gt; that represents a class of recoverable errors that may be thrown during communication between endpoints and which robust <ph id=\"ph1\">[!INCLUDE[indigo1](~/add/includes/ajax-current-ext-md.md)]</ph> client and service applications are expected to handle.","pos":[0,313],"source":"This exception derives from <xref:System.ServiceModel.CommunicationException> that represents a class of recoverable errors that may be thrown during communication between endpoints and which robust [!INCLUDE[indigo1](~/add/includes/ajax-current-ext-md.md)] client and service applications are expected to handle."},{"content":"To prevent the more generic &lt;xref:System.ServiceModel.CommunicationException&gt; handler from catching the more specific &lt;xref:System.ServiceModel.ActionNotSupportedException&gt;, catch this exception prior to handling &lt;xref:System.ServiceModel.CommunicationException&gt;.","pos":[314,577],"source":" To prevent the more generic <xref:System.ServiceModel.CommunicationException> handler from catching the more specific <xref:System.ServiceModel.ActionNotSupportedException>, catch this exception prior to handling <xref:System.ServiceModel.CommunicationException>."}]},{"pos":[2088,2196],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.ServiceModel.ChannelTerminatedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class."},{"pos":[2881,3020],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.ServiceModel.ChannelTerminatedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class with a specified error message."},{"pos":[3032,3189],"content":"Use this constructor when you want to pass an error message to the user. The content of the message parameter should be understandable to the intended users.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"Use this constructor when you want to pass an error message to the user. The content of the message parameter should be understandable to the intended users.","pos":[0,157],"nodes":[{"content":"Use this constructor when you want to pass an error message to the user.","pos":[0,72]},{"content":"The content of the message parameter should be understandable to the intended users.","pos":[73,157]}]}]},{"pos":[3344,3405],"content":"The error message that explains the reason for the exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]},{"pos":[4252,4423],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.ServiceModel.ChannelTerminatedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with serialization information and streaming context specified.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class with serialization information and streaming context specified."},{"pos":[4435,4547],"content":"This constructor is called during deserialization to reconstruct the exception object transmitted over a stream.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"This constructor is called during deserialization to reconstruct the exception object transmitted over a stream.","pos":[0,112]}]},{"pos":[4827,4962],"content":"The <bpt id=\"p1\">&lt;xref href=\"System.Runtime.Serialization.SerializationInfo\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> that contains all the data required to serialize the exception.","needQuote":true,"needEscape":true,"source":"The <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> that contains all the data required to serialize the exception."},{"pos":[5058,5171],"content":"The <xref:System.Runtime.Serialization.StreamingContext> that specifies the source and destination of the stream.","needQuote":true,"needEscape":true,"nodes":[{"content":"The &lt;xref:System.Runtime.Serialization.StreamingContext&gt; that specifies the source and destination of the stream.","pos":[0,113],"source":"The <xref:System.Runtime.Serialization.StreamingContext> that specifies the source and destination of the stream."}]},{"pos":[5840,6053],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.ServiceModel.ChannelTerminatedException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of the exception.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of the exception."},{"pos":[6066,6740],"content":"Use this constructor when you want to pass an error message and information about an inner exception up to the user. The content of the message parameter should be understandable to the intended users.       An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a `null` reference (`Nothing` in Visual Basic) if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.","needQuote":false,"needEscape":true,"extradata":"MT","nodes":[{"content":"Use this constructor when you want to pass an error message and information about an inner exception up to the user.","pos":[0,116]},{"content":"The content of the message parameter should be understandable to the intended users.","pos":[117,201]},{"content":"An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.","pos":[208,387],"source":"       An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property."},{"content":"The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or a <ph id=\"ph1\">`null`</ph> reference (<ph id=\"ph2\">`Nothing`</ph> in Visual Basic) if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.","pos":[388,672],"source":" The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a `null` reference (`Nothing` in Visual Basic) if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor."}]},{"pos":[6922,6983],"content":"The error message that explains the reason for the exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]},{"pos":[7057,7132],"content":"The <xref:System.Exception> that caused the current exception to be thrown.","needQuote":true,"needEscape":true,"nodes":[{"content":"The &lt;xref:System.Exception&gt; that caused the current exception to be thrown.","pos":[0,75],"source":"The <xref:System.Exception> that caused the current exception to be thrown."}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.ServiceModel.ChannelTerminatedException\n  id: ChannelTerminatedException\n  children:\n  - System.ServiceModel.ChannelTerminatedException.#ctor\n  - System.ServiceModel.ChannelTerminatedException.#ctor(System.String)\n  - System.ServiceModel.ChannelTerminatedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.ServiceModel.ChannelTerminatedException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: ChannelTerminatedException\n  nameWithType: ChannelTerminatedException\n  fullName: System.ServiceModel.ChannelTerminatedException\n  type: Class\n  summary: This exception is typically thrown on the client when a channel is terminated due to server closing the associated connection.\n  remarks: This exception derives from <xref:System.ServiceModel.CommunicationException> that represents a class of recoverable errors that may be thrown during communication between endpoints and which robust [!INCLUDE[indigo1](~/add/includes/ajax-current-ext-md.md)] client and service applications are expected to handle. To prevent the more generic <xref:System.ServiceModel.CommunicationException> handler from catching the more specific <xref:System.ServiceModel.ActionNotSupportedException>, catch this exception prior to handling <xref:System.ServiceModel.CommunicationException>.\n  syntax:\n    content: 'public class ChannelTerminatedException : System.ServiceModel.CommunicationException'\n  inheritance:\n  - System.ServiceModel.CommunicationException\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor\n  id: '#ctor'\n  parent: System.ServiceModel.ChannelTerminatedException\n  langs:\n  - csharp\n  name: ChannelTerminatedException()\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException()\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException()\n  type: Constructor\n  assemblies:\n  - System.ServiceModel\n  namespace: System.ServiceModel\n  summary: Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class.\n  syntax:\n    content: public ChannelTerminatedException ();\n    parameters: []\n  overload: System.ServiceModel.ChannelTerminatedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.ServiceModel.ChannelTerminatedException\n  langs:\n  - csharp\n  name: ChannelTerminatedException(String)\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException(String)\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException(String)\n  type: Constructor\n  assemblies:\n  - System.ServiceModel\n  namespace: System.ServiceModel\n  summary: Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class with a specified error message.\n  remarks: Use this constructor when you want to pass an error message to the user. The content of the message parameter should be understandable to the intended users.\n  syntax:\n    content: public ChannelTerminatedException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The error message that explains the reason for the exception.\n  overload: System.ServiceModel.ChannelTerminatedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.ServiceModel.ChannelTerminatedException\n  langs:\n  - csharp\n  name: ChannelTerminatedException(SerializationInfo,StreamingContext)\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException(SerializationInfo,StreamingContext)\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.ServiceModel\n  namespace: System.ServiceModel\n  summary: Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class with serialization information and streaming context specified.\n  remarks: This constructor is called during deserialization to reconstruct the exception object transmitted over a stream.\n  syntax:\n    content: protected ChannelTerminatedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The <xref href=\"System.Runtime.Serialization.SerializationInfo\"></xref> that contains all the data required to serialize the exception.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The <xref:System.Runtime.Serialization.StreamingContext> that specifies the source and destination of the stream.\n  overload: System.ServiceModel.ChannelTerminatedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.ServiceModel.ChannelTerminatedException\n  langs:\n  - csharp\n  name: ChannelTerminatedException(String,Exception)\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException(String,Exception)\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException(String,Exception)\n  type: Constructor\n  assemblies:\n  - System.ServiceModel\n  namespace: System.ServiceModel\n  summary: Initializes a new instance of the <xref href=\"System.ServiceModel.ChannelTerminatedException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of the exception.\n  remarks: \"Use this constructor when you want to pass an error message and information about an inner exception up to the user. The content of the message parameter should be understandable to the intended users.  \\n  \\n An exception that is thrown as a direct result of a previous exception can include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor, or a `null` reference (`Nothing` in Visual Basic) if the <xref:System.Exception.InnerException%2A> property does not supply the inner exception value to the constructor.\"\n  syntax:\n    content: public ChannelTerminatedException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: The error message that explains the reason for the exception.\n    - id: innerException\n      type: System.Exception\n      description: The <xref:System.Exception> that caused the current exception to be thrown.\n  overload: System.ServiceModel.ChannelTerminatedException.#ctor*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.ServiceModel.CommunicationException\n  isExternal: false\n  name: System.ServiceModel.CommunicationException\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor\n  parent: System.ServiceModel.ChannelTerminatedException\n  isExternal: false\n  name: ChannelTerminatedException()\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException()\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException()\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor(System.String)\n  parent: System.ServiceModel.ChannelTerminatedException\n  isExternal: false\n  name: ChannelTerminatedException(String)\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException(String)\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.ServiceModel.ChannelTerminatedException\n  isExternal: false\n  name: ChannelTerminatedException(SerializationInfo,StreamingContext)\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException(SerializationInfo,StreamingContext)\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor(System.String,System.Exception)\n  parent: System.ServiceModel.ChannelTerminatedException\n  isExternal: false\n  name: ChannelTerminatedException(String,Exception)\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException(String,Exception)\n  fullName: System.ServiceModel.ChannelTerminatedException.ChannelTerminatedException(String,Exception)\n- uid: System.Exception\n  parent: System\n  isExternal: true\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.ServiceModel.ChannelTerminatedException.#ctor*\n  parent: System.ServiceModel.ChannelTerminatedException\n  isExternal: false\n  name: ChannelTerminatedException\n  nameWithType: ChannelTerminatedException.ChannelTerminatedException\n"}